router/transformer/transformer.go

```diff
@@ -266,39 +251,56 @@ func (trans *HandleT) ProxyRequest(ctx context.Context, responseData integration
 }
 
 // is it ok to use same client for network and transformer calls? need to understand timeout setup in router
-func (trans *HandleT) Setup() {
+func (trans *HandleT) Setup(netClientTimeout time.Duration) {
 	trans.logger = pkgLogger
 	trans.tr = &http.Transport{}
 	trans.client = &http.Client{Transport: trans.tr, Timeout: timeoutDuration}
+	trans.tfProxyTimeout = 2 * netClientTimeout
```

Comment:
How about `<timeout used in processor stage> + <destination timeout>`?

Reply:
We should also split both the timeout values based on the above comment.

Reply:
It is fine to split the timeout into `x + destination_timeout`. The thing is, I feel this `x` can be a constant or some dynamic value. Why can't this `x` be `destination_timeout` ? Do you feel the timeout will be too high or something ?!

Reply:
`x` should be independent of destinations. If we want to increase the timeout for a given destination to say `1 min`, we'd inadvertently increase the server->transformer timeout also to `1 min`, which may not be acceptable.

Reply:
The changes have been made. Can you verify ?